config/data_centers/data_centers.json

```diff
@@ -679,6 +679,36 @@
     "source": "https://docs.aws.amazon.com/global-infrastructure/latest/regions/aws-regions.html",
     "operationalSince": "2021"
   },
+  "hlrs-de-stuttgart": {
+    "provider": "hlrs",
+    "displayName": "HLRS Stuttgart",
+    "zoneKey": "DE-BW",
+    "lonlat": [9.0970622, 48.739457],
+    "region": "de-stuttgart",
+    "source": "https://www.hlrs.de/",
+    "operationalSince": null,
+    "status": "operational"
+  },
+  "data-center-stuttgart-vaihingen-1": {
```

Comment:
This key is not valid.
It should be a combination of the provider and region the data center owner uses.

Reply:
fixed it